### ISSUE TITLE
4.10.0 Post-release checklist

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#3428]
These are chores that we should do before starting the cycle to reduce ripple effects if we do it in mid-cycle.

Tips: 

- Clean your repo before start
- Remove `node_modules` from all folder
   - `git clean -fdx`
- Never delete `package-lock.json`
- If you mess it up, tableflip and redo
- In `component/package.json`
   - Remove reference to `botframework-webchat-core` by hand-modifying `package.json`
   - Then, `npm install` (symlinks will be broken afterward)
   - Then, add those references back by hand-modifying `package.json`
   - This also applies for other packages with similar dependencies/symlinks
   - To build afterward, do tableflip to rebuild those symlinks

## Applies to all releases

> This list should be copied to versions in the future.

- [x] Bump `package.json` to `4.10.1-0`
   - The PR number is https://github.com/microsoft/BotFramework-WebChat/pull/3433
- [x] ~On QFE branch, make sure `CHANGELOG.md` is cherry-picked to `master`~
- [x] Update [`servicingPlan.json`](https://github.com/microsoft/BotFramework-WebChat/blob/master/packages/embed/servicingPlan.json)
   - Add deprecation notes for previous versions
   - Subresource integrity hash from https://github.com/microsoft/BotFramework-WebChat/releases/tag/v4.10.0
   - The PR is #XXX
- [x] Update all samples to use `4.10.0`
   - Some samples are pointing to GitHub Releases because the sample need new features from daily build
   - Search "https://github.com/microsoft/BotFramework-WebChat/releases/download/"

## Applies to major/minor releases

> Do not applies to patch releases, i.e. 4.10.0 -> 4.10.1.

### Bump all dependencies to latest version

On each `package.json` under root and `/packages/`:

- [x] All NPM packages (`@babel/*`, `jest`, `lerna`, `typescript`, `webpack`, etc) (#XXX)
   - Don't bump the following packages
      - `react`, `react-dom`, `redux`, `redux-saga`
         - Locking down for app compatibility
      - `remark` and `strip-markdown`
         - Newer `remark` don't work under IE11
         - Please go to the their repos and see if this is fixed or by design
      - `serve`
      - `rxjs` on `playground`
      - `microsoft-cognitiveservices-speech-sdk`
- [ ] Run `npm audit fix` to make sure everything is fixed
- [x] Test under IE11 to make sure all dependencies are working

### Bump Docker image

> The Docker image can be found at root `docker-compose.yml` and `Dockerfile*`.

- [ ] Docker container for headless Chrome (#XXX)
   - They recently moved from `3.14.159-xxx` tag scheme to a more sensible `84.0` tag scheme
   - Tags can be found at https://hub.docker.com/r/selenium/node-chrome/tags
   - Preferably in separate PR because screenshots change can be large occasionally
   - Run tests locally, as the screenshots can be slightly different

## Additional steps for 4.10.0

- [x] Clean up branches on official repo
   - We should only have these branches:
      - `master` (or `main`)
      - `gh-pages`
      - `ibiza`
      - `preview-*` (this is for private preview of features)
      - `qfe*`
      - `v3`
   - For 4.10.0 only: please keep `v0.15.0` and investigate later, I am unsure who created this and why it is there.
   - Everyone should fork and submit PR thru their fork
   - Please ping devs who created branches on the official repo about deletion of their branches

**Comment by corinagum** [comment]
Partial fix by PR #3442

**Comment by compulim** [comment]
We will fix it on post-4.10.1.
